crates/krun-guest/src/net.rs

```diff
@@ -28,7 +28,7 @@ pub fn configure_network() -> Result<()> {
     };
     if let Some(dhcpcd_path) = dhcpcd_path {
         let output = Command::new(dhcpcd_path)
-            .args(["-M", "--nodev", "eth0"])
+            .args(["-M", "--nodev", "eth0", "-e", "resolvconf=does-not-exist"])
```

Comment:
By the way, we have `systemd-udevd` now, right? Can you test if it works without `--nodev`?

Reply:
It does, but needs a few more udev commands. Added those.